Assets/HoloToolkit/Input/Scripts/Utilities/MicrophoneHelper.cs

```diff
@@ -0,0 +1,157 @@
+﻿
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+// This sample code is a modified version of an official UWP sample which can be found on:
+// https://github.com/Microsoft/Windows-universal-samples/blob/master/Samples/SpeechRecognitionAndSynthesis/cs/AudioCapturePermissions.cs
+//
+//*********************************************************
+
+#if ENABLE_WINMD_SUPPORT
+using System;
+using System.Threading.Tasks;
+using Windows.Media.Capture;
+#endif
+
+namespace HoloToolkit.Unity
```

Comment:
Why is this class included in your PR?

Reply:
I've changed the target branch for this PR

Reply:
Hmm. I only meant to include my two commits (the inputSource script, followed by the example scene). Indeed, I can only see my four files included my view of the PR..?

Reply:
No need to create a new PR. I've already fixed this.